index.cjs

```diff
@@ -165,6 +165,24 @@ module.exports = {
     // Warn on restricted syntax
     'no-restricted-syntax': [
       'warn',
+      {
+        selector:
+          "ExpressionStatement CallExpression[callee.object.name='document'][callee.property.name='querySelector']",
+        message:
+          'Using `document.querySelector()` can give unexpected results in between renders or if there are multiple elements that match the query (for example, when the component is used more than once), [learn more](https://github.com/reactjs/reactjs.org/issues/4626#issuecomment-1117535930)',
```

Comment:
are we allowed to use Markdown links in ESLint messages?

Reply:
lol I am not sure ill check 
but I think so.

Reply:
[273c877](https://github.com/upleveled/eslint-config-upleveled/pull/105/commits/273c877c5bfa3581e21d34e9436fdd045fd64af3)